.rubocop.yml

```diff
@@ -1,12 +1,59 @@
+require:
+  - rubocop-rake
+  - rubocop-rspec
+  - rubocop-performance
+
 AllCops:
-  TargetRubyVersion: 2.5
+  TargetRubyVersion: 2.7
+  NewCops: enable
   Exclude:
     - '_build/**/*'
     - 'pkg/**/*'
     - 'Gemfile'
     - 'Puppetfile'
     - 'Rakefile'
 
+Style/StringConcatenation:
+  Enabled: true
+  Exclude:
+    - 'hooks/boot/10-reset_data.rb'
+
+Style/SlicingWithRange:
+  Enabled: false
+
+Performance/Casecmp:
```

Comment:
Why disable these performance cops? Many of these look more like something you'd do in a TODO file.

The file `.rubocop.yml` is what you use as a set of rules that you've agreed on. The file `.rubocop_todo.yml` is the file for rules that currently don't fit the guidelines, but that you want to eventually.

Reply:
Previously, I disabled the `Performance/Casecmp` cop due to safety concerns as mentioned in the documentation (https://docs.rubocop.org/rubocop-performance/cops_performance.html#performancecasecmp). Upon reviewing the code changes when enabling this cop, it seems to be safe in this case.

I also observed that enabling the cop resulted in a `Style/NegatedIf` offense, which I disabled to ensure consistency in our code style.